app/components/homepage-components/featured-project-card-layout/ProjectCardImage.tsx

```diff
@@ -16,9 +18,8 @@ const ProjectCardImage = ({
       }`}
     >
       <Image
-        src={imageSrc}
+        src={imageSrc || mobileAndDesktopFallback}
```

Comment:
Adding a fallback image almost defeats the purpose of putting the primary image inside Sanity. Maybe store the image in your public folder? Just thinking...